packages/core/src/connectors/aliyun-dm/index.ts

```diff
@@ -1,6 +1,13 @@
 import { ConnectorType } from '@logto/schemas';
 
-import { ConnectorMetadata } from '../types';
+import {
+  ConnectorError,
+  ConnectorMetadata,
+  EmailMessageTypes,
+  EmailSendMessageFunction,
+} from '../types';
+import { getConnectorConfig } from '../utilities';
+import { sendSingleMail } from './send-single-mail';
 
 export const metadata: ConnectorMetadata = {
```

Comment:
nit: maybe put it into a const config file, along with the connector domain name?

Reply:
The outside don't need to know the `domain name`. Metadata here is meant for necessary info for Logto core.